### DISCUSSION TITLE
Unable to open Storyboard file when using MVVMCross template

### DISCUSSION THREAD

**Discussion by joegreen690** [MvvmCross/MvvmCross#4370] [Q&A]
I'm creating a Xamarin.Forms cross-platform app using MVVM Cross Platform Single Page template. I'm using Visual Studio for Mac and have installed MvvmCross Template Pack from VS Extension Manager. When I double click on storyboard file in Visual Studio, I'm expecting the file to open in Xcode Interface Builder. Xcode is open but I get an error that says -- The project ‘HelloCrossPlatformWorld.iOS’ is damaged and cannot be opened due to a parse error. Examine the project file for invalid edits or unresolved source control conflicts.

Next error message appears which says Visual Studio could not communicate with Xcode.

I'm new to MVVM Cross and it hasn't been a smooth road. Like some posts said to remove extension and reinstall the template pack which I have done but still getting the same errors. I've uninstalled and reinstalled Xcode.

Any idea what is causing these errors? Is it time to try another MVVM framework or just forget about these MVVM frameworks?

**Comment by Cheesebaron** [comment]
> Is it time to try another MVVM framework or just forget about these MVVM frameworks?

Perhaps it is with this attitude.
